Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'json'
+gem 'ruby-beautify'
```

Comment:
Up til now, we've been able to keep `cfndsl` free of outside dependencies. Is there a way to make this work without an additional gem?